src/test/java/com/puppycrawl/tools/checkstyle/MainTest.java

```diff
@@ -302,14 +324,31 @@ public void testExistingTargetFileWithError()
             throws Exception {
         exit.expectSystemExitWithStatus(2);
         exit.checkAssertionAfterwards(() -> {
+            final LocalizedMessage auditStartMessage = new LocalizedMessage(0,
+                    Definitions.CHECKSTYLE_BUNDLE, "DefaultLogger.auditStarted",
+                    null, null, getClass(), null);
+            final LocalizedMessage auditFinishMessage = new LocalizedMessage(0,
+                    Definitions.CHECKSTYLE_BUNDLE, "DefaultLogger.auditFinished",
+                    null, null, getClass(), null);
```

Comment:
@rnveach , I do not like class fields for Tests, but I think that moving auditStartMessage/auditFinishMessage and other string without parameters to Test classfields is reasonable.